### DISCUSSION TITLE
Playback for MPEGTS (.ts) Files?

### DISCUSSION THREAD

**Discussion by piyushpranjal03** [bluenviron/mediamtx#4084] [Questions]
### Question

Hi eveyone,

When I was recording my videos in `fmp4` format it is getting recording with 1sec lag. As recorded videos were having 1sec lag so I switched to `mpegts` now the recording is perfect but now I can't use the playback feature. I don't want to install any separate video player.

Is there any way to use the playback for .ts files?

Thanks in advance!

**Comment by aler9** [comment]
this is being tracked in #3054 - however, keep in mind that browsers are not able to play MPEG-TS files, therefore an external player is still required, either a full software installed on your machine (i.e. VLC) or a JavaScript library (mpegts.js).
